### PR TITLE
Update i2c.md

### PR DESCRIPTION
**Pull request by kiate**
Added EDU:BIT i2c address in micro:bit accessories table.

### CODE REVIEW

File: hardware/i2c.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Overall, this means 0x1D, 0x0E (from v1.3), 0x1E and 0x19 (for the revision) are
 | accessory name | organisation | i2c address(es) used | 
 |----------------|--------------|-----------------------|
 | eg [banana-bit]()|eg Banana enterprises | eg 0x76, 0x29 |
+| [EDU:BIT](https://www.cytron.io/p-edubit)| Cytron Technologies | 0x08 |
```
Comment (microbit-carlos) [inline hardware/i2c.md:37]: Thanks @kiate, is this a 7-bit address (excludes the read/write bit)?

Comment (kiate) [reply hardware/i2c.md:37]: @carlosperate Yes, that's right.

Comment (microbit-carlos) [reply hardware/i2c.md:37]: Great, in that case could you provide the addresses in the same format as the micro:bit sensors? `0x__ (0x__/0x__)`

@microbit-mark we should update the example to follow this pattern as well.

Comment (kiate) [reply hardware/i2c.md]: Done!

File: hardware/i2c.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ Overall, this means 0x1D, 0x0E (from v1.3), 0x1E and 0x19 (for the revision) are
 | accessory name | organisation | i2c address(es) used | 
 |----------------|--------------|-----------------------|
 | eg [banana-bit]()|eg Banana enterprises | eg 0x76, 0x29 |
```
Comment (microbit-mark) [inline hardware/i2c.md:36]: I think we can just remove the example now we have a real world entry : )
```suggestion
```

Comment (microbit-carlos) [reply hardware/i2c.md:36]: Sounds good too 👍

**Comment by microbit-mark** [comment]
Thanks for your contribution @kiate

**Comment by kiate** [comment]
Thanks, Mark & Carlos!

**Pull request merged**